src/current/v25.1/alter-table.md

```diff
@@ -1572,13 +1570,6 @@ The [TPC-C]({% link {{ page.version.version }}/performance-benchmarking-with-tpc
 
 To change the data type from `DECIMAL` to `STRING`:
 
-1. Set the `enable_experimental_alter_column_type_general` [session variable]({% link {{ page.version.version }}/set-vars.md %}) to `true`:
-
-    {% include_cached copy-clipboard.html %}
-    ~~~ sql
-    > SET enable_experimental_alter_column_type_general = true;
-    ~~~
-
 1. Alter the column type:
 
     {% include_cached copy-clipboard.html %}
```

Comment:
This comment should be a few lines down for this line:
> NOTICE: ALTER COLUMN TYPE changes are finalized asynchronously; further schema changes on this table may be restricted until the job completes; some writes to the altered column may be rejected until the schema change is finalized

We don't return the NOTICE to the user anymore. This is the legacy schema changer implementation. This notice is repeated a couple of times on this page.

Reply:
removed all instances of this NOTICE